app/src/main/java/nl/mpcjanssen/simpletask/Simpletask.kt

```diff
@@ -954,10 +933,9 @@ class Simpletask : ThemedNoActionBarActivity() {
             val currentIntent = getIntent()
             currentIntent.putExtra(SearchManager.QUERY, intent.getStringExtra(SearchManager.QUERY))
             setIntent(currentIntent)
-            if (options_menu == null) {
-                return
+            options_menu?.let {
+                it.findItem(R.id.search).collapseActionView()
             }
-            options_menu!!.findItem(R.id.search).collapseActionView()
```

Comment:
Brittle

Reply:
I am not sure i prefer misbehaviour over a crash here

Reply:
The point is scoping, so an incorrect refactor would fail to compile instead of failing at runtime, not behavior change.

That said, I realized this actually does change the behavior, because I dropped the 'return'. I've pushed a commit to fix that. I can change it to throw an exception instead of returning, if you'd like.